index.js

```diff
@@ -1181,7 +1181,7 @@ export const createJavaBom = async (path, options) => {
             `maven settings.xml found in ${basePath}. Please set the MVN_ARGS environment variable based on the full mvn build command used for this project.\nExample: MVN_ARGS='--settings ${settingsXml}'`
           );
         }
-        const mavenCmd = getMavenCommand(basePath, path);
+        const mavenCmd = '"'+ getMavenCommand(basePath, path)+'"';
```

Comment:
Prabhu, I have checked the changes with double back slashes, still I am getting same kind of issue, attached some images  for windows, please refer those.

![cdxgen-path-issue](https://github.com/CycloneDX/cdxgen/assets/144091669/4c9bf818-2603-4a5e-99e6-87f18f6e4142)

Reply:
How is the value specified? Via env args? Can you escape space with a single backslash?

Reply:
This value is passed via env args and did some hardcoded for testing purpose.
As well `path.resolve()` returns path with single slash for os portable. 
A single backslash (\) cannot escape a space.

Here are some additional notes from ChatGPT:

- Double backslashes are usually used in Windows paths, while single backslashes are more common in other operating systems.
- Using double backslashes can make our code less portable, as it may not work on all platforms.

Reply:
Update:
Adding maven path to system environment variable is working fine.
When we add a environmental variable MAVEN_HOME for user specific then it throws above error.

Reply:
Even in WSL we are getting this space issue.